catalog/CHANGELOG.md

```diff
@@ -16,4 +16,4 @@ where verb is one of
 
 ## Changes
 
-- [Added] Bootstrap the change log ([#XXXX](https://github.com/quiltdata/quilt/pull/XXXX))
```

Comment:
oh shi~ i forgot to insert the actual PR number 🤦 
but let's keep that entry anyways (might as well fix it)
could you please bring it back and set the PR number to 4112?